### PR TITLE
[release/8.0-preview5] Bump package version for .net 7 to 7.0.6

### PR DESCRIPTION
**Pull request by steveisok**

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -7,12 +7,17 @@
...
+               set $(WorkloadsTestPreviousVersions)=false, so those manifests won't be installed
+               during testing. -->
+    <PackageVersionNet7>7.0.6</PackageVersionNet7>
     <PackageVersionNet6>6.0.$([MSBuild]::Add($([System.Version]::Parse('$(PackageVersionNet7)').Build),11))</PackageVersionNet6>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>5</PreReleaseVersionIteration>
     <WorkloadVersionSuffix Condition="'$(PreReleaseVersionLabel)' != 'release'">-$(PreReleaseVersionLabel).$(PreReleaseVersionIteration)</WorkloadVersionSuffix>
     <SdkBandVersionForWorkload_FromRuntimeVersions>$(SdkBandVersion)$(WorkloadVersionSuffix)</SdkBandVersionForWorkload_FromRuntimeVersions>
+    <!-- set to false for release branches -->
+    <WorkloadsTestPreviousVersions Condition="'$(WorkloadsTestPreviousVersions)' == ''">false</WorkloadsTestPreviousVersions>
```
Comment (ViktorHofer) [inline eng/Versions.props:20]: Why not set this based on `'$(DotNetFinalVersionKind)' == 'servicing'` which is defined below?

Comment (steveisok) [reply eng/Versions.props:20]: @lewing is there a servicing scenario where we *can* run WBT?

Comment (lewing) [reply eng/Versions.props:20]: in theory we can run them if PackageVersionNet7 <= netcoreapp.LatestRuntimeFrameworkVersion

**Comment by carlossanlop** [comment]
Let me know if this is good to merge or if any more commits will be added to address the unresolved conversation.

File: eng/Versions.props (hunk 1 above)
Comment (steveisok) [reply eng/Versions.props:20]: @ViktorHofer not sure we can do anything right now.  Best to leave it as is and think more about it.

**Comment by steveisok** [comment]
> Let me know if this is good to merge or if any more commits will be added to address the unresolved conversation.

It's good to go.

**Comment by lewing** [comment]
https://github.com/dotnet/emsdk/pull/347 should go in if this does (they should match)

**Comment by lewing** [comment]
ping @marcpopMSFT

**Pull request merged**